build-tools/libzip/libzip.targets

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <_SourceTopDir>..\..</_SourceTopDir>
+  </PropertyGroup>
+  <UsingTask AssemblyFile="$(_SourceTopDir)\bin\Build$(Configuration)\Xamarin.Android.Tools.BootstrapTasks.dll" TaskName="Xamarin.Android.Tools.BootstrapTasks.GetNugetPackageBasePath" />
+  <Import Project="libzip.props" />
+  <Import Project="libzip.projitems" />
+  <Target Name="_Make"
+        Inputs=""
```

Comment:
No inputs means `make` will always be executed.

As suggested earlier (which I can't find?!), you should follow [mono-runtimes.targets](https://github.com/xamarin/xamarin-android/blob/master/build-tools/mono-runtimes/mono-runtimes.targets#L61-L75):

1. Have a target, e.g. `_SetConfigureAcTimeToLastCommitTimestamp`, which uses `touch -m` to set the timestamp on e.g. `configure.ac` to the timestamp of the last commit in that repo.
2. Add `_SetConfigureAcTimeToLastCommitTimestamp` to the `DependsOnTargets` of the `_Make` target.
3. Depend on e.g. `configure.ac` as an `Inputs` to the `_Make` target.

The timestamp + touch computation are (reasonably) quick, whereas invoking `make` might not be. (For example, top-level `make` within `monodroid` is *slow*, even when there's nothing to do!)